python/services/metadata_service.py

```diff
@@ -0,0 +1,109 @@
+import requests
+import logging
+
+
+class MetadataService:
+    def __init__(self, api_url: str, api_token: str):
+        self.api_url = api_url
+        self.api_token = api_token
+
+    def get_acceptable_slack_users(self):
+        return [
+            {"username": "sam.pepper"},
+            {"username": "connor.glynn"}
+        ]
+
+    def get_acceptable_github_usernames(self):
+        return [
+            {"username": "PepperMoJ"},
+            {"username": "connormaglynn"}
+        ]
+
+    def get_existing_slack_users(self):
+        try:

```

Comment:
Think we could omit the try/catch here? 🤔 Just let the exception bubble up and allow the caller to decide if they want to handle it 🤹 

I think in our case, we wouldn't want to handle the exception - and we'd be happy to let the script fail if it can't talk to Slack for some reason 🔥 Would also mean we could add the Slack alerts for this scenario in the Github action 💌